model/table.go

```diff
@@ -88,14 +88,16 @@ func (t *table) Options() chan TableOption {
 	return ch
 }
 
-// NewTableOption creates a new table option with the given name and value
-func NewTableOption(k, v string) TableOption {
+// NewTableOption creates a new table option with the given name and value and quarts
```

Comment:
`name and value and quarts` → `name, value, and a flag indicating if quoting is necessary`